packages/react/src/AvatarStack/AvatarStack.tsx

```diff
@@ -249,6 +260,11 @@ const AvatarStack = ({
     )
   }
 
+  useEffect(() => {
+    const hasInteractive = getInteractiveNodes(AvatarStackContainer.current, '[tabindex]')
+    setHasInteractiveChildren(hasInteractive)
+  }, [children])
```

Comment:
Would this be equivalent to:

```tsx
useEffect(() => {
  // ...
})
```

I wasn't sure if `children` would change each render or not if they are not memoized. If that's the case, would it be worth running as a mutation observer, potentially?

Reply:
We could! My assumption is that we'd want to utilize the mutation observer inside of `hasInteractiveNodes`?

Reply:
I see, this is great! Thank you for the example too. I'll rework this a bit.

Reply:
After implementing this, I realize that the need for checking the children for changes is probably very slim 😅 I removed the `children` dependency so it should only run on the initial render, as that's the only time we'd actually want to run this function/check.